PlayCover/ViewModel/StoreVM.swift

```diff
@@ -84,8 +109,6 @@ class StoreVM: ObservableObject {
     }
 
     func resolveSources() {
-        if !NetworkVM.isConnectedToNetwork() { return }
```

Comment:
Don't remove this check

Reply:
Ah, I might've accidentally deleted that during merge.